### DISCUSSION TITLE
How does JSBSim handle multiple aircraft interactions under the hood?

### DISCUSSION THREAD

**Discussion by Chulabhaya** [JSBSim-Team/jsbsim#995] [Q&A]
Hi all! I have a bit of a simple question which is what I'm wondering how it works under the hood when I instantiate multiple aircraft through several lines of `fdm0 = jsbsim.FGFDMExec(None)`. Is it that the first call will instantiate a sort of global simulator where any subsequent instantiations of `fdm1 = jsbsim.FGFDMExec(None), fdm2 = jsbsim.FGFDMExec(None)` will initialize the other aircraft within that global simulator? I just want to better visualize in my head what's happening when I initialize several different FDMs in terms of how they would interact.

**Comment by seanmcleod** [comment]
They would all be independent.

**Reply by Chulabhaya** [reply]
Understood, thanks!
